jellyfish/_room_api.py

```diff
@@ -60,27 +67,33 @@ def create_room(
         In such case, a new `RoomApi` instance has to be created using the returned address
         in order to interact with the room.
         """
+
+        if video_codec is not None:
+            video_codec = RoomConfigVideoCodec(video_codec)
+        else:
+            video_codec = None
+
         room_config = RoomConfig(
-            maxPeers=max_peers, videoCodec=video_codec, webhook_url=webhook_url
+            max_peers=max_peers, video_codec=video_codec, webhook_url=webhook_url
         )
-        resp = self._room_api.create_room(room_config)
 
+        resp = self._request(create_room, json_body=room_config)
         return (resp.data.jellyfish_address, resp.data.room)
 
     def delete_room(self, room_id: str) -> None:
         """Deletes a room"""
 
-        return self._room_api.delete_room(room_id)
+        return self._request(delete_room, room_id=room_id)
```

Comment:
What do you think about passing here function as `room.delete_room` instead of `delete_room`,  because for some time, I asked myself if this is a recursive function.